### PR TITLE
Established roles

### PR DESCRIPTION
**Pull request by SoniaHL**
Added a Home page with 2 buttons. Currently both buttons take you to a page that displays the bookstore inventory.

### CODE REVIEW

File: .idea/dataSources.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
+    <data-source source="LOCAL" name="sample" uuid="60bba5d2-8dbc-4dc5-ab29-57e16bb68344">
+      <driver-ref>sqlite.xerial</driver-ref>
+      <synchronize>true</synchronize>
+      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
+      <jdbc-url>jdbc:sqlite:C:\Users\SONIA\git\SYSC4806Project\sample.db</jdbc-url>
+      <working-dir>$ProjectFileDir$</working-dir>
+    </data-source>
+  </component>
+</project>
```
Comment (mounicap6) [inline .idea/dataSources.xml:12]: Do we have to push this file?

File: src/main/java/com.amazon.bookstore/BookstoreController.java [hunk 2]
```diff
@@ -7,8 +7,8 @@
 @Controller
 public class BookstoreController {
 
-    @GetMapping("/")
-    public String index(Model model) {
```
Comment (mounicap6) [inline src/main/java/com.amazon.bookstore/BookstoreController.java:11]: If this index method is removed, we have to create an index.html page that our program goes to when there is an error (a fallback page).

File: src/main/java/com.amazon.bookstore/BookstoreController.java [hunk 3]
```diff
@@ -7,8 +7,8 @@
 @Controller
 public class BookstoreController {
```
Comment (mounicap6) [inline src/main/java/com.amazon.bookstore/BookstoreController.java:8]: Do we have to test controllers?

File: .idea/dataSources.xml (hunk 1 above)
Comment (SoniaHL) [reply .idea/dataSources.xml:12]: You're right. Don't think we pushed this before. Will delete it from the PR

File: src/main/java/com.amazon.bookstore/BookstoreController.java (hunk 2 above)
Comment (SoniaHL) [reply src/main/java/com.amazon.bookstore/BookstoreController.java:11]: What if I make homePage() the index() method instead? That way it falls back to the very beginning

Comment (mounicap6) [reply src/main/java/com.amazon.bookstore/BookstoreController.java:11]: That sounds like it should work

**Review by sarahlamonica — APPROVED** [review]

**Pull request merged**